src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationAnalysisChartPresenter.cs

```diff
@@ -15,25 +13,30 @@
 using OSPSuite.Presentation.Services.Charts;
 using OSPSuite.Presentation.Views;
 using OSPSuite.Presentation.Views.ParameterIdentifications;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Extensions;
 
 namespace OSPSuite.Presentation.Presenters.ParameterIdentifications
 {
-   public abstract class ParameterIdentificationAnalysisChartPresenter<TChart, TView, TPresenter> : SimulationAnalysisChartPresenter<TChart, TView, TPresenter>, IParameterIdentificationAnalysisPresenter
+   public abstract class
+      ParameterIdentificationAnalysisChartPresenter<TChart, TView, TPresenter> : SimulationAnalysisChartPresenter<TChart, TView, TPresenter>,
```

Comment:
Let's put the name on the same line as the class . This looks just too weird

Reply:
done